app/styles/_substructure.less

```diff
@@ -79,17 +79,20 @@ html,body {
         }
         .middle-content {
           .flex(@columns: 1 1 auto);
-          // default 50px space beneath content
-          padding-bottom: @middle-content-bottom-margin;
           position: relative;
           width: 100%;
+          > .container-fluid {
+            // default 50px space beneath content
+            padding-bottom: @middle-content-bottom-margin;
+          }
         }
       }
     }
   }
-  & .overview .middle .middle-section .middle-container .middle-content {
-      // reduce 20px to account for overview tiles bottom margin
-      padding-bottom: (@middle-content-bottom-margin - 20px);
+  & .overview .middle .middle-section .middle-container .middle-content > .container-fluid,
```

Comment:
This is extremely specific. Does it need to be? It makes it hard to change the structure of the page or override styles when we string so many class selectors together.